cmd/SplitGrantsGovXMLDB/handler_test.go

```diff
@@ -53,9 +51,8 @@ func (m mockDDBClientGetItemCollection) NewGetItemClient(t *testing.T) mockDynam
 		err := attributevalue.UnmarshalMap(params.Key, &getItemKey)
 		require.NoError(t, err, "Failed to extract grant_id value from DynamoDB GetItem key")
 		output := dynamodb.GetItemOutput{Item: nil}
-		targetGrantId, exists := getItemKey["grant_id"]
 		var rvErr error
-		if exists {
+		if targetGrantId, exists := getItemKey["grant_id"]; exists {
```

Comment:
This is just a change in variable grouping that I noticed as a readability issue; it has nothing to do with the rest of the PR.